### ISSUE TITLE
mirror broken with latest aframe master threejs r163

### ISSUE THREAD

**Issue by vincentfretin** [mrxz/fern-aframe-components#1]
Using 
```html
<script src="https://cdn.jsdelivr.net/npm/@fern-solutions/aframe-mirror@1.1.0/dist/mirror.umd.min.js"></script>
<script src="https://cdn.jsdelivr.net/gh/aframevr/aframe@27f0308c49902008fd5307a863da790b890df2d9/dist/aframe-master.min.js"></script>
```
latest build build from Apr 1, 2024 with three r162, it works great.
![image](https://github.com/mrxz/fern-aframe-components/assets/112249/507b91fb-c49e-492d-bc75-30671329309d)

With newer build from Apr 1, 2024 with three r163
```html
<script src="https://cdn.jsdelivr.net/gh/aframevr/aframe@3e7aa71a486e064fb7bd23e11799cea588d4a662/dist/aframe-master.min.js"></script>
```
![image](https://github.com/mrxz/fern-aframe-components/assets/112249/5da3b108-79aa-4721-9ab1-ea0b3360afa9)

**Comment by vincentfretin** [comment]
You can remix https://glitch.com/edit/#!/naf-valid-avatars?path=public%2Findex.html%3A13%3A81 if you need to reproduce the issue.

**Comment by mrxz** [comment]
Thanks for reporting! This seems to be due to the fact that Three.js used no longer has the stencil buffer enabled by default in r163. I've created a PR for A-Frame to add an option that allows enabling it: https://github.com/aframevr/aframe/pull/5509

I'll see if I can add a warning message to the mirror component in the meantime to warn users if the stencil buffer is disabled.

**Comment by mrxz** [comment]
Should be fixed on the latest A-Frame master with https://github.com/aframevr/aframe/pull/5509 merged. Also released a new version 1.1.1 that warns the user if `stencil: true` isn't set on the `renderer` system (and sets it itself, if not).

Let me know if that fixes it for you.

**Comment by vincentfretin** [comment]
Oh wow you were quick. Thanks, it works great now.
